server/src/main/java/io/unitycatalog/server/auth/decorator/UnityAccessUtil.java

```diff
@@ -0,0 +1,60 @@
+package io.unitycatalog.server.auth.decorator;
+
+import com.auth0.jwt.interfaces.Claim;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import io.unitycatalog.control.model.User;
+import io.unitycatalog.server.auth.UnityCatalogAuthorizer;
+import io.unitycatalog.server.exception.BaseException;
+import io.unitycatalog.server.model.Privilege;
+import io.unitycatalog.server.persist.MetastoreRepository;
+import io.unitycatalog.server.persist.UserRepository;
+import io.unitycatalog.server.persist.model.CreateUser;
+import io.unitycatalog.server.service.AuthDecorator;
+import java.util.UUID;
+
+public class UnityAccessUtil {
+
+  private static final UserRepository USER_REPOSITORY = UserRepository.getInstance();
+  private static final MetastoreRepository METASTORE_REPOSITORY = MetastoreRepository.getInstance();
+
+  public static UUID findPrincipalId() {
+    ServiceRequestContext ctx = ServiceRequestContext.current();
+    DecodedJWT decodedJWT = ctx.attr(AuthDecorator.DECODED_JWT_ATTR);
+    // TODO: if/when authorization becomes mandatory, maybe just throw an exception here?
+    if (decodedJWT != null) {
+      Claim sub = decodedJWT.getClaim("sub");
```

Comment:
could we replace with JwtClaim.SUBJECT

Reply:
Done. There were several places. And added an `EMAIL` claim too.